src/LndClient/Data/PayReq.hs

```diff
@@ -45,15 +45,16 @@ addSeconds =
     . fromRational
     . toRational
     . secondsToDiffTime
-    . fromIntegral
+    . from
```

Comment:
Our convention for usage of Witch - to always specify types when from/tryFrom is used. Maybe I should create a linter on innovation time, haha. Not a big problem, but specify types if that is not difficult for you.